File: src/main/java/com/spring/dto/response/AlbumResponse.java
package com.spring.dto.response;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.util.List;

@Data
@Builder
@AllArgsConstructor
@NoArgsConstructor
public class AlbumResponse {
    private Long id;
    private String albumName;
    private String description;
    private String releaseDate;
    private Float albumTimeLength;
    private List<Long> songIdList;
    private List<String> songNameList;
    private List<Long> artistIdList;
    private List<String> artistNameList;
    private String imageUrl;
    private String status;
}

File: MSMA/app/src/main/java/vn/edu/usth/msma/data/dto/response/management/AlbumResponse.kt
package vn.edu.usth.msma.data.dto.response.management

import com.google.gson.annotations.SerializedName

data class AlbumResponse(
    @SerializedName("id") val id: Long,
    @SerializedName("albumName") val albumName: String,
    @SerializedName("description") val description: String,
    @SerializedName("releaseDate") val releaseDate: String,
    @SerializedName("albumTimeLength") val albumTimeLength: Float,
    @SerializedName("songIdList") val songIdList: List<Long>,
    @SerializedName("songNameList") val songNameList: List<String>,
    @SerializedName("artistIdList") val artistIdList: List<Long>,
    @SerializedName("artistNameList") val artistNameList: List<String>,
    @SerializedName("imageUrl") val imageUrl: String,
    @SerializedName("status") val status: String
)